servers/zts/src/main/java/com/yahoo/athenz/zts/notification/CertFailedRefreshNotificationTask.java

```diff
@@ -160,7 +160,7 @@ public CertFailedRefreshNotificationTask(InstanceCertManager instanceCertManager
             }
 
             String expiryTime =  getTimestampAsString(certRecord.getExpiryTime());
-            String hostName = (certRecord.getHostName() != null) ? certRecord.getHostName() : "";
```

Comment:
At this point we already check that the record is valid and so it could never be nulll